vendor/bat-native-ledger/src/bat_get_media.cc

```diff
@@ -511,11 +511,39 @@ void BatGetMedia::processYoutubeMediaPanel(uint64_t windowId,
     processYoutubeChannelPath(windowId, visit_data, providerType);
   } else if (visit_data.path.find("/user/") != std::string::npos) {
     processYoutubeUserPath(windowId, visit_data, providerType);
+  } else if (!isYTPath(visit_data.path)) {
+    processYoutubeCustomPath(windowId, visit_data, providerType);
   } else {
     onMediaActivityError(visit_data, providerType, windowId);
   }
 }
 
+bool BatGetMedia::isYTPath(const std::string& path) const {
```

Comment:
maybe `isPredifinedYTPath`? Current name is a little bit confusing

Reply:
Done